src/modules/function/declaration.rs

```diff
@@ -272,12 +275,43 @@ impl DocumentationModule for FunctionDeclaration {
     fn document(&self, meta: &ParserMetadata) -> String {
         let mut result = vec![];
         result.push(format!("## `{}`", self.name));
+
+        let exe_path = env::current_exe().expect("Error executable not found").parent().expect("Error executable path not found").display().to_string();
+        let mut test_reference = String::from("You can check the original tests for code examples: ");
+        let mut found_test = false;
+        if exe_path.contains("target/debug") {
+            let file_std = meta.context.path.as_ref().and_then(|p| Path::new(&p).file_name()?.to_str().map(|s| s.to_string())).unwrap_or_default();
+            result.push(format!("\n```import {{ {} }} from \"std/{}\"```\n", self.name, file_std));
+            let std_test_path = exe_path.replace("target/debug", "src/tests/stdlib");
+            let std_test_path_buf = Path::new(&std_test_path);
+            if std_test_path_buf.exists() && std_test_path_buf.is_dir() {
+                if let Ok(entries) = fs::read_dir(std_test_path_buf) {
+                    for entry in entries.flatten() {
+                        let path = entry.path();
+                        if let Some(file_name) = path.file_name().and_then(|f| f.to_str()) {
+                            if glob::Pattern::new(&format!("{}*.ab", self.name)).unwrap().matches(file_name) {
```

Comment:
I do have some Rust suggestions; but first we need to resolve a design issue.  I tried running your code on `array.ab`:
```
hwalters@Ghostwheel ~/git/amber (doc-tests-ii) 
$ ./target/debug/amber --docs src/std/array.ab
 INFO  File generated at:
/home/hwalters/git/amber/src/std/docs/array.md
```
This adds text to the markdown file as expected:
```
hwalters@Ghostwheel ~/git/amber (doc-tests-ii) 
$ egrep -w 'code examples' src/std/docs/array.md
You can check the original tests for code examples: [array_first_index.ab](https://github.com/amber-lang/amber/blob/master/src/tests/stdlib/array_first_index.ab).
You can check the original tests for code examples: [array_search.ab](https://github.com/amber-lang/amber/blob/master/src/tests/stdlib/array_search.ab).
You can check the original tests for code examples: [includes_partial_match_with_expanded_element.ab](https://github.com/amber-lang/amber/blob/master/src/tests/stdlib/includes_partial_match_with_expanded_element.ab).
```
Your code tests for globs `array_first_index*.ab`, `array_search*.ab` and `includes*.ab`. Unfortunately there are multiple matches for `includes*.ab`, so it picks the first one:
```
hwalters@Ghostwheel ~/git/amber (doc-tests-ii) 
$ ls -1 src/tests/stdlib/includes*.ab
src/tests/stdlib/includes_empty_num_array.ab
src/tests/stdlib/includes_empty_text_array.ab
src/tests/stdlib/includes_exact_match.ab
src/tests/stdlib/includes_partial_match_with_expanded_element.ab
src/tests/stdlib/includes_prefix_match.ab
src/tests/stdlib/includes_text_array.ab
```
This needs a bit of a rethink.  For example:
1. Should we include all six files matching `includes*.ab`?
2. Can we guarantee that all relevant files will start with `includes`?
3. Can we guarantee that any irrelevant files will not start with `includes`?